roles/postfix/README.md

```diff
@@ -0,0 +1,35 @@
+Role Name
+=========
+
+Creates a relay to proofpoint and creates a relay for our applications
+
+
+Role Variables
+--------------
+
+Most use for this role will involve adding your IP address to the allow list with the following variable:
+
+```bash
+relay_new_host: 1.2.3.4
```

Comment:
This var name might get confusing. At the time that we add a new host, it would make sense. A year later, it might not feel "new". Not sure how to handle this . . . could we have a central list of "IPs that should have access to our email relay"?

Reply:
heh. var names are ALWAYS hard for me! 😬